### PR TITLE
Guilherme

### PR DESCRIPTION
**Pull request by GuilhermeGuimaraesDeMedeiros**
Task8_1
Task8_2

### CODE REVIEW

File: dist/cows/C3PO.cow [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# C3PO
```
Comment (flaviostutz) [inline dist/cows/C3PO.cow]: The dist folder shouldn't be pushed to the repo

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+run-slug:
+	npm run build && node src/slug.js
+
+test:
+	npm run test
+
+cow-slug:
+	pnpm run cow-slug
+
```
Comment (flaviostutz) [inline Makefile:9]: The command for making the cow say a random phrase should be part of a .ts file, or, at most, present in Makefile so others could use it.

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Change the target branch to your branch in my repo, not to main.l 😄

File: Makefile (hunk 2 above)
Comment (GuilhermeGuimaraesDeMedeiros) [reply Makefile:9]: Command included in Makefile 
say:
   pnpm run say

File: dist/cows/C3PO.cow (hunk 1 above)
Comment (GuilhermeGuimaraesDeMedeiros) [reply dist/cows/C3PO.cow]: dist folder included in the .gitignore

Comment (flaviostutz) [reply dist/cows/C3PO.cow]: Yes, but you need to delete it from the repo now

File: Makefile (hunk 2 above)
Comment (flaviostutz) [reply Makefile:9]: Is this command outputting the cow saying a random word? Could you post a print screen of the result of this command?

**Review by flaviostutz — CHANGES_REQUESTED** [review]
I left some comments 😁

File: dist/cows/C3PO.cow (hunk 1 above)
Comment (GuilhermeGuimaraesDeMedeiros) [reply dist/cows/C3PO.cow]: Done

File: Makefile (hunk 2 above)
Comment (GuilhermeGuimaraesDeMedeiros) [reply Makefile:9]: Captura de tela 2024-11-24 115905

Comment (GuilhermeGuimaraesDeMedeiros) [reply Makefile:9]: The command 'pnpm run say' outputs a cow saying "I´m a moooodule". The command 'pnpm run cow-slug' outputs a cow saying a random word. Task 8 asks to create a package script called 'cow-slug' that, when run, showa the cow saying a random slug. Therefore, I maintained the 'say' script created in Task 5 and created the 'cow-slug' script to output the cow saying a random word.

**Review by flaviostutz — APPROVED** [review]
Nice!!

**Pull request merged**